src/sentry/monitors/logic/incident_occurrence.py

```diff
@@ -144,6 +145,11 @@ def send_incident_occurrence(
     if last_successful_checkin:
         last_successful_checkin_timestamp = last_successful_checkin.date_added.isoformat()
 
+    detector = get_detector_for_monitor(monitor_env.monitor)
+    evidence_data = {}
+    if detector:
```

Comment:
Not sure if I should also put this behind a feature flag, but I think it's not likely to cause many issues. What do you think @saponifi3d?

Reply:
right now this should be fine because the `post_process` is only setup for error events and metric issues.

we will need issue alerts to start responding to more than just error events soon -- but we'll add an allow/deny list when we go to do that so it's easy to enable/disable different issue types.